### PR TITLE
Introduce the Media Reference Stack

### PR DESCRIPTION
**Pull request by lsandov1**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ services:
   - docker
 
 env: 
-  #- IMAGE_NAME=stacks_dlrs_mkl DIR=dlrs/clearlinux/tensorflow/mkl/
-  - IMAGE_NAME=stacks_dlrs_oss DIR=dlrs/clearlinux/tensorflow/oss/
+  - IMAGE_NAME=stacks_dlrs_oss DIR=dlrs/clearlinux/tensorflow/oss/ BUILD_ARGS='--build-arg clear_ver="30650"'
```
Comment (DnPlas) [inline .travis.yml]: Let's add this change in a different PR if needed, we should be updating these dockerfiles soon, so BUILD_ARGS might not be necessary at all in the future.
+1 on the mers changes

File: mers/clearlinux/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Media Reference Stack
```
Comment (DnPlas) [inline mers/clearlinux/README.md:1]: Let's add a microbadger banner on this readme

File: mers/clearlinux/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+> **Note:**
+    Since Clear Linux OS is a stateless system, avoid modifying the
+    files under the `/usr` directory instead use `/etc` otherwise the software
+    updater may overwrite `/usr` files.
+
+## Run examples
+
+For transcoding and video analytics examples and pipelines, please see the
+official documentation at: https://clearlinux.org/sw/stacks/media-reference
```
Comment (DnPlas) [inline mers/clearlinux/README.md:49]: I guess this link will work once we go live, right?

File: mers/clearlinux/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# Media Reference Stack
+
+## Building Locally
+
+The Dockerfiles for all Intel System Stacks container images are available at [stacks repository](https://github.com/intel/stacks). These can be used to build and modify the container images.
```
Comment (DnPlas) [inline mers/clearlinux/README.md]: Can we add a paragraph that explicitly says the following command is for building the image?

File: .travis.yml (hunk 1 above)
Comment (rahulunair) [reply .travis.yml]: +1

**Comment by rahulunair** [comment]
@lsandov1  let's split the travis builds and add it separately

File: mers/clearlinux/README.md (hunk 4 above)
Comment (lsandov1) [reply mers/clearlinux/README.md]: yes

**Comment by lsandov1** [comment]
@DnPlas @rahulunair sounds good. PR is just about MeRS, not touching the travis file. Thanks

**Comment by DnPlas** [comment]
LGTM

**Pull request merged**